src/procedures/EnablePercentageTransferManager.ts

```diff
@@ -38,12 +48,33 @@ export class EnablePercentageTransferManager extends Procedure<
       moduleName,
     });
 
-    await this.addTransaction<TransactionParams.SecurityToken.AddPercentageTransferManager>(
-      securityToken.addModuleWithLabel,
-      {
-        tag: PolyTransactionTag.EnablePercentageTransferManager,
-      }
-    )({
+    const [newPtmAddress] = await this.addTransaction<
+      TransactionParams.SecurityToken.AddPercentageTransferManager,
+      [string]
+    >(securityToken.addModuleWithLabel, {
+      tag: PolyTransactionTag.EnablePercentageTransferManager,
+      resolvers: [
+        async receipt => {
```

Comment:
This resolver should be refactored into a separate function to be testable (and tested)

Reply:
yes I know. It will be the next task in the list. I just rise this PR to be sure the logic is ok to go forward